index.js

```diff
@@ -24,43 +22,64 @@ function parseBody (req, res, next) {
   })
 }
 
-var router = Router()
+// Injected dependencies
+exports['@require'] = [ 'commands' ]
```

Comment:
This is a dependency annotation. This will recursively resolve `src/commands` dependencies and inject it into `index.js`